src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/DroolSscsJudicialOfficeMappingTest.java

```diff
@@ -25,53 +25,6 @@
 @RunWith(MockitoJUnitRunner.class)
 class DroolSscsJudicialOfficeMappingTest extends DroolBase {
 
-    //=================================SALARIED ROLES==================================
```

Comment:
Rather than retired can this be either split or adjusted so JOH tests remain for :

- President of Tribunal,Salaried
- Regional Tribunal Judge,Salaried
- Tribunal Judge,Salaried

Reply:
> added back in 74c13e5. only removed work types check, as there is already a thorough work type check in `DroolSscsJudicialRoleMappingTest`